internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -741,6 +741,11 @@ func resourceVirtualMachineScaleSet() *pluginsdk.Resource {
 							Optional: true,
 						},
 
+						"automatic_upgrade_enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+						},
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```